### DISCUSSION TITLE
Manuscript project including slides

### DISCUSSION THREAD

**Discussion by jburos** [quarto-dev/quarto-cli#10767] [Feature Requests]
### Description

In the course of drafting a paper or scholarly artifact, I commonly want to showcase the work to others. This may be a status report for internal stakeholders or lab-mates, or a talk at a conference or proceeding. 

It would be _really_ nice if I could keep these presentation-ready slides along with my manuscript & notebooks in the quarto project for a manuscript. Bonus if the slides can cross-link notebooks for figures and tables, which could allow me to keep a paradigm of notebooks as source.

Right now, the notebooks are too technical to support this use case, and I have hit odd walls trying to render any formats beyond what the `manuscript` project expects. As a work-around, I am either editing the _quarto.yml file to remove the `project type: manuscript` lines while working on the slides, then adding them back after rendering (if it's an old project repo), or setting up the manuscript part as a subfolder of the repository to enable other formats elsewhere in the folder hierarchy.

Posting here to offer food-for-thought, in case you haven't considered this use case. I love where manuscripts is going. Hope this is helpful.

**Comment by davidzgli** [comment]
This would be a nice feature.  I often have the same situation where I need to present my work while writing the manuscript.

**Comment by mcanouil** [comment]
Why not use profiles? https://quarto.org/docs/projects/profiles.html

This is mostly in response to:

> As a work-around, I am either editing the _quarto.yml file to remove the project type: manuscript lines while working on the slides, then adding them back after rendering (if it's an old project repo), or setting up the manuscript part as a subfolder of the repository to enable other formats elsewhere in the folder hierarchy.

**Reply by davidzgli** [reply]
Project profile seems to be promising.  Will test it out.

**Comment by harrelfe** [comment]
Here are some easy-to-do compromise approaches.

It is easy to add slide-like executive summaries at the start of a detailed document as I have done [here](https://hbiostat.org/bbr/propensity).  You can also easily hide these with a collapsible note.

Adding `mermaid` or `graphviz` road maps to presentations is also easy as shown [here](https://hbiostat.org/bayes/bet/multiplicity).  A little further down you'll see something that approximates real slides by using `tabsets` to reveal one slide at a time.
